joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstCreator.scala

```diff
@@ -70,6 +77,7 @@ class AstCreator(val relPathFileName: String, val parserResult: ParserResult)
       case BasicLit                                 => astForLiteral(nodeInfo)
       case Ident                                    => astForIdentifier(nodeInfo)
       case FuncDecl                                 => astForFuncDecl(nodeInfo)
+      case _: BaseExprStmt                          => astForExpression(nodeInfo)
```

Comment:
Does this not work simply by matching to `BaseExprStmt`?

Reply:
No, because `BaseExprStatement` is a `sealed trait` others are `object` here